src/planning/nav2_planning_tests/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 3.5)
+project(nav2_planning_tests)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall) # -Wextra) # -Wpedantic)
+endif()
+
+find_package(rclcpp REQUIRED)
+find_package(nav2_tasks REQUIRED)
+find_package(nav2_util REQUIRED)
+find_package(nav2_libs_msgs REQUIRED)
+find_package(nav2_planning_msgs REQUIRED)
+find_package(nav2_world_model_msgs REQUIRED)
+find_package(nav_msgs REQUIRED)
+find_package(visualization_msgs REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
+find_package(ament_cmake_pytest REQUIRED)
+
+set(TEST_LAUNCH_DIR ${CMAKE_CURRENT_SOURCE_DIR}/test_launch_files)
+
+ament_add_gtest_executable(test_planner_node
```

Comment:
All these cmake commands to build and add the tests should probably go in the `BUILD_TESTING` section. It means there won't be anything to build if that flag is not set, but I think that is correct behavior.

Reply:
Fixed.